test/integration/dialects/postgres/connection-manager.test.js

```diff
@@ -47,6 +47,13 @@ if (dialect.match(/^postgres/)) {
       const error = await sequelize.query('select pg_sleep(2)').catch(e => e);
       expect(error.message).to.equal('Query read timeout');
     });
+
+    it('should allow overriding session variables through the `options` param', async () => {
```

Comment:
I somehow expected a test that would use a database url as you proposed in the description

Reply:
I would like to add a test like that - but I wasn't sure where before. i just found it in `test/integration/sequelize.test.js` now. I'll add a test for `options` there, but I'm having trouble finding any sort of SSL testing anywhere.